src/catkin_pkg/changelog_generator_vcs.py

```diff
@@ -179,14 +179,19 @@ def get_latest_tag_name(self):
         tag_name = result_describe['output']
         return tag_name
 
-    def get_log_entries(self, from_tag, to_tag, skip_merges=False):
+    def get_log_entries(self, from_tag, to_tag, skip_merges=False, only_merges=False):
         # query all hashes in the range
+        MSG_ERR_OPTIONS_MERGE_CONTRADICT = 'Both "skip_merges" and "only_merges" are set to True, which contradicts.'
```

Comment:
Please avoid a single use local variable and instead move the string literal to where it is used below.